assembl/fabfile.py

```diff
@@ -1401,8 +1404,10 @@ def update_node(force_reinstall=False):
         # Stop gulp and webpack because otherwise node may be busy
         supervisor_process_stop('dev:gulp')
         supervisor_process_stop('dev:webpack')
+        venvcmd("rm -rf venv/lib/node_modules/")
         venvcmd("rm -f venv/bin/npm")  # remove the symlink first otherwise next command raises OSError: [Errno 17] File exists
-        venvcmd("nodeenv --node=6.11.5 --npm=3.10.10 --python-virtualenv assembl/static/js")
+        venvcmd("nodeenv --node=10.13.0 --npm=6.4.1 --python-virtualenv assembl/static/js")
+        execute(upgrade_yarn)
         with cd(get_node_base_path()):
             venvcmd("npm install reinstall -g", chdir=False)
         with cd(get_new_node_base_path()):
```

Comment:
instead of 
```
with cd(get_node_base_path()):
    venvcmd("npm install reinstall -g", chdir=False)
with cd(get_new_node_base_path()):
    venvcmd("npm install reinstall -g", chdir=False)  # which is wrong because this doesn't use yarn for static2
```
you should use 
```
update_npm_requirements(True)
```